Know-Your-Nation-Speedy/Know-Your-Nation-Speedy/Models/Organisation.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Know_Your_Nation_Speedy.Models
+{
+    public class Organisation
+    {
+        public Organisation()
+        {
+            Initialise();
+        }
+
+        public int Id { get; set; }
+        public string Name { get; set; }
+        public string Description { get; set; }
+        public string URL { get; set;}
```

Comment:
Usually its better practice to make URLs of type `uri`

Reply:
Thanks didnt know about this